test/commands/project/deploy/pipeline/resume.test.ts

```diff
@@ -177,6 +188,28 @@ describe('project deploy pipeline resume', () => {
         expect(monitorStub.called).to.equal(true);
       });
 
+    test
+      .stdout()
+      .stderr()
+      .do(() => {
+        mockAorRecord = {
+          Id: mockAorId,
+          sf_devops__Message__c: 'mockMessage',
+          sf_devops__Status__c: AsyncOperationStatus.InProgress,
+          sf_devops__Error_Details__c: 'mockErrorDetail',
+        };
+        sandbox.stub(AorSelector, 'selectAsyncOperationResultById').resolves(mockAorRecord);
+
+        monitorStub = sinon.stub(AsyncOpStreaming.prototype, 'monitor');
+        sandbox.stub(ResumeCommandOutputService.prototype, 'getStatus').returns(AsyncOperationStatus.Completed);
+        stubDisplayEndResults = sandbox.stub(ResumeCommandOutputService.prototype, 'displayEndResults');
+      })
+      .command(['project deploy pipeline resume', `-i=${mockAorId}`, '--verbose'])
```

Comment:
How about a test to check that stubDisplayEndResults is not called when the verbose flag is not used? And another one for the concise case (the output should be the same as if it was not used)?